src/Microsoft.AspNetCore.Antiforgery/AntiforgeryOptions.cs

```diff
@@ -14,81 +14,78 @@ public class AntiforgeryOptions
+        public string CookieName { get => Cookie.Name; set => Cookie.Name = value; }
 
         /// <summary>
-        /// This is obsolete and will be removed in a future version.
-        /// The recommended alternative is to use ConfigureCookieOptions.
+        /// <para>
+        /// This property is obsolete and will be removed in a future version. The recommended alternative is the <seealso cref="CookieBuilder.Path"/> on <see cref="Cookie"/>.
+        /// </para>
+        /// <para>
         /// The path set on the cookie. If set to <c>null</c>, the "path" attribute on the cookie is set to the current
         /// request's <see cref="HttpRequest.PathBase"/> value. If the value of <see cref="HttpRequest.PathBase"/> is
         /// <c>null</c> or empty, then the "path" attribute is set to the value of <see cref="CookieOptions.Path"/>.
+        /// </para>
         /// </summary>
-        [Obsolete("This is obsolete and will be removed in a future version. The recommended alternative is to use ConfigureCookieOptions.")]
-        public PathString? CookiePath { get; set; }
+        [Obsolete("This property is obsolete and will be removed in a future version. The recommended alternative is " + nameof(Cookie) + "." + nameof(CookieBuilder.Path) + ".")]
+        public PathString? CookiePath { get => Cookie.Path; set => Cookie.Path = value; }
 
         /// <summary>
-        /// This is obsolete and will be removed in a future version.
-        /// The recommended alternative is to use ConfigureCookieOptions.
+        /// <para>
+        /// This property is obsolete and will be removed in a future version. The recommended alternative is the <seealso cref="CookieBuilder.Domain"/> on <see cref="Cookie"/>.
+        /// </para>
+        /// <para>
         /// The domain set on the cookie. By default its <c>null</c> which results in the "domain" attribute not being set.
+        /// </para>
         /// </summary>
-        [Obsolete("This is obsolete and will be removed in a future version. The recommended alternative is to use ConfigureCookieOptions.")]
-        public string CookieDomain { get; set; }
+        [Obsolete("This property is obsolete and will be removed in a future version. The recommended alternative is " + nameof(Cookie) + "." + nameof(CookieBuilder.Domain) + ".")]
+        public string CookieDomain { get => Cookie.Domain; set => Cookie.Domain = value; }
 
         /// <summary>
-        /// Configures the <see cref="CookieOptions"/> of the antiforgery cookies. Without additional configuration, the 
-        /// default values antiforgery cookie options are true for <see cref="CookieOptions.HttpOnly"/>, null for
-        /// <see cref="CookieOptions.Domain"/> and <see cref="SameSiteMode.Strict"/> for <see cref="CookieOptions.SameSite"/>.
+        /// Determines the settings used to create the antiforgery cookies.
         /// </summary>
-        public Action<HttpContext, CookieOptions> ConfigureCookieOptions { get; set; }
```

Comment:
I know we did this to enable per-request configuration of things like the path, but I can't find an actual customer request for this, so I think we can just forget it until someone asks. Like you said, build is virtual, so we could use our subclass to resurrect this if necessary